update-v4-readmes.sh

```diff
@@ -1,38 +1,43 @@
 #!/bin/bash
 
-# The following is a 5 column list of org, repo, branch, package, and file.
-# - If the branch is NOT specified, then the README for that project
-#   will be pulled from npmjs.org instead and will reflect the latest
-#   release.
+# The following is a 5 column list of org, repo, branch, file, and module.
+# - If the module is specified, the README for that project will be
+#   pulled from npmjs.org instead and will reflect the latest release.
+#   For scoped packages such as `@loopback/metadata`,  the `/` needs to
+#   be encoded as `%2f`. For example `@loopback%2fmetadata`.
 # - If the branch IS specified, it will be used to fetch the README.md
 #   from the given github repo. If that branch is NOT master, then the
 #   branch name will be appended to the local readme file name.
+#
+# Examples:
+# strongloop loopback-next master packages/metadata/README.md
+# strongloop loopback-next master packages/metadata/README.md @loopback%2fmetadata
+#
 (cat <<LIST_END
-strongloop loopback-next master metadata README.md
+strongloop loopback-next master packages/metadata/README.md
```

Comment:
Does this need to be altered to include all of the packages? It seems like we're only pointing at metadata with this setup.

Reply:
For LB4, we only pull in README for `metadata` at this moment. This is the place you can add more readmes to be imported.